docs/spec/channel.md

```diff
@@ -187,13 +187,19 @@ exclusively communicate using CloudEvents.
 #### Input
 
 Every Channel MUST expose either an HTTP or HTTPS endpoint. It MAY expose both.
-The endpoint(s) MUST conform to CloudEvents
-[Version 1.0](https://github.com/cloudevents/spec/blob/v1.0/http-protocol-binding.md).
+The endpoint(s) MUST conform to one of the following versions of the specification:
+
+* [CloudEvents 0.2 specification](https://github.com/cloudevents/spec/blob/v0.2/http-transport-binding.md)
```

Comment:
I could've sworn that some of these earlier ones were somewhat problematic, do we really want to start allowing for these older ones as well?

Reply:
I think for backwards compatibilty - we should - I guess at least support 0.3, and of courese 1.0 

Perhpas we do some recommendation for 1.0  and support 0.3 ? (I mean... it works),

and what goes in ... goes out